ideascube/mediacenter/tests/test_views.py

```diff
@@ -620,3 +620,27 @@ def test_can_update_document_tags(staffapp, pdf):
     doc = Document.objects.get(pk=pdf.pk)
     assert doc.tags.count() == 2
     assert doc.tags.first().name == 'tag1'
+
+
+def test_content_summary_is_not_cleaned_from_wanted_html_tags(staffapp):
+    url = reverse('mediacenter:document_create')
+    form = staffapp.get(url).forms['model_form']
+    form['title'] = 'my content title'
+    form['summary'] = 'my content summary <a href="foo">link</a>'
+    form['credits'] = 'my document credits'
+    form['original'] = Upload('image.jpg', b'xxxxxx', 'image/jpeg')
+    form.submit().follow()
+    document = Document.objects.first()
+    assert document.summary == ('my content summary <a href="foo">link</a>')
```

Comment:
> There is no javascript here, so no tinyMCE.

I know, that was a tongue-in-cheak comment, made right after I said something about the migration wrapping things in `<p>`. 😛 

I probably should have made it clearer that it was just a joke, or just refrained from writing this at all.

I don't think you need to wrap it in a `<p>` here, it really doesn't matter.